mypy/test/testtypes.py

```diff
@@ -1560,7 +1560,7 @@ def make_call(*items: tuple[str, str | None]) -> CallExpr:
 class TestExpandTypeLimitGetProperType(TestCase):
     # WARNING: do not increase this number unless absolutely necessary,
     # and you understand what you are doing.
-    ALLOWED_GET_PROPER_TYPES = 9
+    ALLOWED_GET_PROPER_TYPES = 10
```

Comment:
I don't understand what I'm doing! This is probably bad, please approach the relevant code (in `expandtype.py`) with skepticism; this is for not causing `*args: Unpack[tuple[Unpack[tuple[...]]]]` because mypy doesn't expect nested Unpacks like that (and neither do my changes in this PR).